dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -281,6 +305,10 @@ private void logFinalSummary(
                   + state.getTasksReports().stream()
                       .map(taskReport -> taskReport.count() + " " + taskReport.state())
                       .collect(joining(", ")));
+          // For now, it will return true only for TeradataLogsConnector and Terada14LogsConnector
```

Comment:
Why doesn't it return `true` for all Logs Connectors?

Reply:
Because for now I implemented shared state logic only for Teradata. I can make all log connectors return `true`, it will work because of this line `if (QueryLogSharedState.queryLogEntries.size() == 0) {
      return;
    }` in `outputFirstAndLastQueryLogEnries` method. Just wanted to be consistent with main logic for this change.

Reply:
It should be consistent with the connector output type - if it retrieves query logs, then a method called `isLogConnector` should return `true`, in all other cases it should be `false`. I think the simplest fix is to rename the method.
For example, if no extension is planned for the future, then `isTeradataLogConnector` would be better, otherwise `shouldOutputFirstAndLastQueryLog` is an option.

Reply:
Potential extension is planned for other connectors. So if we generally go with this approach, then in future might be implementing for others as well. So in that case `shouldOutputFirstAndLastQueryLog` naming would be more suitable. Will change to that.